packages/shared/src/__tests__/telemetry.test.ts

```diff
@@ -0,0 +1,155 @@
+import 'cross-fetch/polyfill';
```

Comment:
`fetch` isn't available in the `jsdom` jest environment 😢, so adding the polyfill here.